node/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
+    "co-body": "^6.0.0",
+    "ramda": "^0.25.0"
+  },
+  "devDependencies": {
+    "@vtex/tsconfig": "^0.4.4",
+    "@types/co-body": "^0.0.3",
+    "@types/jest": "^24.0.18",
+    "@types/node": "^12.0.0",
+    "@types/ramda": "types/npm-ramda#dist",
```

Comment:
What's this?

Reply:
I actually don't know. It came with the project service-example that I used as a starting point